feathr_project/feathr/registry/feature_registry.py

```diff
@@ -0,0 +1,42 @@
+from abc import ABC, abstractmethod
+
+from typing import Any, Dict, List, Optional, Tuple
+from feathr.definition.feature_derivations import DerivedFeature
+from feathr.definition.anchor import FeatureAnchor
+
+class FeathrRegistry(ABC):
+    """This is the abstract class for all the feature registries. All the feature registries should implement those interfaces.
+    """
+
+    @abstractmethod
+    def register_features(self, from_context: bool = True):
```

Comment:
I think we should be able to remove 'from_context'?

Reply:
solved